### ISSUE TITLE
Incompatible with Laravel 10

### ISSUE THREAD

**Issue by lstg** [aymanrb/php-unstructured-text-parser#34]
Thanks for your package, it looks very useful :)  

Unfortunately we've been attempting to use it with Laravel 10 but it gets hung up on the following dependencies : 

```
 - Root composer.json requires aymanrb/php-unstricted-text-parser ^2.3 -> satisfiable by aymanrb/php-unstricted-text-parser[v2.3.0].
 - aymanrb/php-unstricted-text-parser v2.3.0 requires psr/log ^1.1 -> found psr/log[1.1.0, …, 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.   
```

```
- monolog/monolog 3.3.1 requires psr/log ^2.0 || ^3.0 found psr/log[2.0.0, 3.0.0] but the package is fixed to 1.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command. 
- laravel/framework v10.11.0 requires monolog/monolog ^3.0 satisfiable by monolog/monolog[3.3.1].
- laravel/framework is locked to version v10.11.0 and an update of this package was not requested.
```

I've bumped the dependency locally and it seems to work, will attach a pull request to correct this for your consideration

**Comment by aymanrb** [comment]
Thanks for reporting the issue and the PR ... it should be fixed as you suggested in v2.4.0 now. Kindly check it and let me know if that works as expected now.

**Comment by lstg** [comment]
Great, thanks, I can confirm that v2.4.0 has resolved the issue :)
